.pylintrc

```diff
@@ -19,7 +19,24 @@ disable = arguments-differ,
           wrong-import-order,
           attribute-defined-outside-init,
           import-outside-toplevel,
-          duplicate-code
+          duplicate-code,
+          arguments-renamed,
+          unspecified-encoding,
+          consider-using-from-import,
+          consider-using-with,
+          metatype,
+          consider-using-enumerate,
+          ref_model_state_dict,
+          saved_coeffs_in_file[key],
+          unsubscriptable-object,
+          node_key_list,
+          unsupported-membership-test,
+          deprecated-decorator,
+          consider-using-dict-items,
+          use-list-literal,
+          use-dict-literal,
+          use-a-generator,
+          use-maxsplit-arg
```

Comment:
Shouldn't we instead fix the corresponding places in code? @alexsu52 is there anyone up for this right now?

Reply:
This is will be the right thing. But  ofc it is your call.